js/huobipro.js

```diff
@@ -32,6 +32,8 @@ module.exports = class huobipro extends Exchange {
                 'fetchMyTrades': true,
                 'withdraw': true,
                 'fetchCurrencies': true,
+                'fetchDeposits': 'emulated',
```

Comment:
I think this property should be `true` because `fetchDeposits` actually is not emulated (same for `fetchWithdrawals`).